debug/test/browser/debug.test.js

```diff
@@ -275,6 +275,15 @@ describe('debug', () => {
 			expect(serializeVNode(<Foo />)).to.equal('<Foo />');
 		});
 
+		it('should serialize vnode without props', () => {
+			const vnode = {
```

Comment:
This kind of `vnode` doesn't exist. Only `vnode`'s representing a text node have `props` set to `null`. That's very likely why the error occurred.

```suggestion
const vnode = {
  type: null,
  props: null,
  text: 'hello world',
  key: null,
  ref: null
};
```

Reply:
Thanks! I was really struggling to figure out how the vnode could end up with null props. A text node didn't really cross my mind. 👍